jd4/case.py

```diff
@@ -19,7 +19,7 @@
 MAX_STDERR_SIZE = 8192
 WAIT_JITTER_NS = 5000000
 PROCESS_LIMIT = 32
-DEFAULT_MEM_KB = 131072
+DEFAULT_MEM_KB = 524288
```

Comment:
I found the value 131072 in vijos-jd

Also note that memory usage in linux will be much smaller than windows because only RSS is included.